index.html

```diff
@@ -195,28 +195,26 @@ <h3 class="connect">LET’S CONNECT</h3>
 
 		<footer>
 			<section class="contact-me">
-				<h1 class="contact-me-text">contact me</h1>
+				<h1 class="contact-me-text">Contact me</h1>
 				<p class="contact-me-brief">If you have an application you are intrested in developing, a feature that
 					you need built or a project that needs coding.
 					I'd love to help with it.
 				</p>
-				<form action="">
+				<form action="https://formspree.io/f/my-portfolio" method="post">
 					<div class="input-field-box">
-						<input class="input-field" type="text" name="NAME" maxlength="30" value="Type your name">
-						<input class="input-field" type="email" name="EMAIL" value="username@domain.ext">
+						<input class="input-field" type="text" name="name" maxlength="30" value="Type your name">
+						<input class="input-field" type="email" name="email" value="username@domain.ext">
 						<textarea class="write-message" name="write message" placeholder="Write your message here"
```

Comment:
- [ ] Kindly remove the `rows` and the `cols` attributes from the textarea and add the `maxlength="500"` attribute also you have to add the `required` attribute.